code/modules/mob/living/carbon/human/species_types/psyphoza.dm

```diff
@@ -0,0 +1,480 @@
+	meat = /obj/item/reagent_containers/food/snacks/meat/slab/human/mutant/psyphoza
+	species_traits = list(NOEYESPRITES, AGENDER, MUTCOLORS)
+	sexes = FALSE
+	changesource_flags = MIRROR_BADMIN | WABBAJACK | MIRROR_MAGIC | MIRROR_PRIDE | ERT_SPAWN | RACE_SWAP
+	species_language_holder = /datum/language_holder/psyphoza
+	exotic_blood = /datum/reagent/drug/mushroomhallucinogen
+	allow_numbers_in_name = TRUE
+	inert_mutation = SPORES
+
+	offset_features = list(OFFSET_UNIFORM = list(0,0), OFFSET_ID = list(0,0), OFFSET_GLOVES = list(0,0), OFFSET_GLASSES = list(0,-2), OFFSET_EARS = list(0,-3), OFFSET_SHOES = list(0,0), OFFSET_S_STORE = list(0,0), OFFSET_FACEMASK = list(0,-2), OFFSET_HEAD = list(0,-2), OFFSET_FACE = list(0,-2), OFFSET_BELT = list(0,0), OFFSET_BACK = list(0,0), OFFSET_SUIT = list(0,0), OFFSET_NECK = list(0,0))
+
+	mutant_brain = /obj/item/organ/brain/psyphoza
+	mutanteyes = /obj/item/organ/eyes/psyphoza
+	mutanttongue = /obj/item/organ/tongue/psyphoza
+
+	mutant_bodyparts = list("psyphoza_cap")
+	default_features = list("psyphoza_cap" = "Portobello", "body_size" = "Normal", "mcolor" = "fff")
+	hair_color = "fixedmutcolor"
+
+	species_chest = /obj/item/bodypart/chest/psyphoza
+	species_head = /obj/item/bodypart/head/psyphoza
+	species_l_arm = /obj/item/bodypart/l_arm/psyphoza
+	species_r_arm = /obj/item/bodypart/r_arm/psyphoza
+	species_l_leg = /obj/item/bodypart/l_leg/psyphoza
+	species_r_leg = /obj/item/bodypart/r_leg/psyphoza
+
+	//Fire bad!
+	burnmod = 1.25
+
+	//Reference to psychic highlight action
+	var/datum/action/item_action/organ_action/psychic_highlight/PH
+
+/datum/species/psyphoza/on_species_gain(mob/living/carbon/C, datum/species/old_species, pref_load)
+	. = ..()
+	PH = locate(/datum/action/item_action/organ_action/psychic_highlight) in C.actions
+	ADD_TRAIT(C, TRAIT_PSYCHIC_SENSE, SPECIES_TRAIT)
+
+/datum/species/psyphoza/on_species_loss(mob/living/carbon/human/C, datum/species/new_species, pref_load)
+	. = ..()
+	PH = null
+
+/datum/species/psyphoza/random_name(gender, unique, lastname, attempts)
+	. = "[pick(GLOB.psyphoza_first_names)] [pick(GLOB.psyphoza_last_names)]"
+	if(unique && attempts < 10 && findname(.))
+		return .(gender, TRUE, null, ++attempts)
+
+/datum/species/psyphoza/handle_chemicals(datum/reagent/chem, mob/living/carbon/human/H)
+	if(istype(chem, /datum/reagent/drug) && H.blood_volume < BLOOD_VOLUME_NORMAL)
+		H.blood_volume += chem.volume * 15
+		H.reagents.remove_reagent(chem.type, chem.volume)
+		return FALSE
+	return ..()
+
+/datum/species/psyphoza/get_scream_sound(mob/living/carbon/user)
+	return pick('sound/voice/psyphoza/psyphoza_scream_1.ogg', 'sound/voice/psyphoza/psyphoza_scream_2.ogg')
+
+/datum/species/psyphoza/primary_species_action()
+	. = ..()
+	PH?.Trigger()
+
+/datum/species/psyphoza/get_species_description()
+	return "Psyphoza are a species of extra-sensory lesser-sensory fungal, \
+	based humanoids, infamous for their invulnerability to occlusion based \
+	magic tricks."
```

Comment:
```suggestion
	return "Psyphoza are a species of blind psychic fungal-form \
	humanoids, infamous for their invulnerability to occlusion-based \
	magic tricks and sleight of hand."
```

Reply:
I like 'extra-sensory lesser-sensory', but the other stuff works